### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by Vicky-ind** [community/community#45613] [GitHub Education]
### Select Topic Area

Question

### Body

have filled out every single spot on my GitHub user profile and it still won't allow me to continue with my [GitHub Student Developer Pack](https://education.github.com/pack) application because!

Your application cannot be reviewed until you fix the following: • You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio., Have you completed your GitHub user profile with all your

relevant information, such as your full name as

it appears in your image and a short bio?

**Comment by jge162** [comment]
Hello @MaggieChristiansen I looked at your profile and you appear to have completed it. Have you submitted a request since updating it? the application from the instructions are very easy to complete. I think you just need to either reapply or login to see if you are approved.

**Reply by jge162** [reply]
Here is a detailed guide to completing the application [GitHub education documentation](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)
